cudamapper/include/claraparabricks/genomeworks/cudamapper/utils.hpp

```diff
@@ -0,0 +1,56 @@
+/*
+* Copyright 2019-2020 NVIDIA CORPORATION.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#pragma once
+
+#include <mutex>
```

Comment:
Is this in reference to the just the <mutex> header? IMO it makes sense to have it here, since `std::mutex` is in the function signature. Am I missing something as to why having the header here isn't necessary?

Reply:
Sorry, you're right, I missed the fact that `mutex` is a parameter